test/integration/targets/collections/import_collection_pb.yml

```diff
@@ -0,0 +1,2 @@
+- import_playbook: testns.testcoll.default_collection_playbook.yml
```

Comment:
This `.yml` ending looks counterintuitive. I'm :-1: on this. We need to be strict.

Reply:
its a thing that was requested early in the feedback, im 50/50 since playbooks already allow for this but there is argument to be made to force being strict in the future. Most authors complain that for proper highlighting editors want the extension.

Reply:
But that's for actual files on disk. What I'm concerned about is how the importable string looks.

Reply:
Just to clarify: it's not about yaml/yml. It's about how the end-users would reference playbooks from other YAML files or CLI.

Reply:
ah, so keep the 'magic matching' but remove the ability to specify the extension? Since we are relying on the python importer to do the work, we would have to explicitly disallow this and filter the extension out (waits for 'my "yml" playbook does not work inside a collection!").